bgpd/bgp_zebra.c

```diff
@@ -64,6 +65,8 @@
 #include "bgpd/bgp_evpn_mh.h"
 #include "bgpd/bgp_mac.h"
 
+#define min(A, B) ((A) < (B) ? (A) : (B))
```

Comment:
mmm - we've got MIN and MAX in `lib/compiler.h` ?

Reply:
Fixed.